sklearn/utils/estimator_checks.py

```diff
@@ -1470,8 +1502,10 @@ def check_estimators_overwrite_params(name, Estimator):
                      % (name, param_name, original_value, new_value))
 
 
+@ignore_warnings(category=DeprecationWarning)
 def check_no_fit_attributes_set_in_init(name, Estimator):
     """Check that Estimator.__init__ doesn't set trailing-_ attributes."""
+    # STILL ON CLASSES
```

Comment:
Once the world of scikit-learn <= 0.18 is forgotten, this comment will look pretty obscure. How about "Takes a class rather than an estimator instance as input"